test_package/conanfile.py

```diff
@@ -110,8 +110,11 @@ def _test_with_meson(self):
     def _test_with_cmake(self):
         if self._cmake_supported():
             self.output.info("Testing CMake")
-            shutil.copy("qt.conf", os.path.join("cmake_folder", "bin"))
-            self.run(os.path.join("cmake_folder", "bin", "test_package"), run_environment=True)
```

Comment:
You can also keep these 2 lines as-is, if you leave the 4 lines highlighted above in test_package/CMakeLists.txt